Formula/neovim.rb

```diff
@@ -3,54 +3,105 @@ class Neovim < Formula
+      sha256 "afda64ca445203bb7092372b822bae8b2539fdcebbfc3f753f393628c2bcfe7d"
+    end
 
-  resource "msgpack" do
-    url "https://github.com/msgpack/msgpack-c/archive/cpp-1.0.0.tar.gz"
-    sha256 "afda64ca445203bb7092372b822bae8b2539fdcebbfc3f753f393628c2bcfe7d"
-  end
+    resource "luajit" do
+      url "http://luajit.org/download/LuaJIT-2.0.4.tar.gz"
+      sha256 "620fa4eb12375021bef6e4f237cbd2dd5d49e56beb414bee052c746beef1807d"
+    end
 
-  resource "luajit" do
-    url "http://luajit.org/download/LuaJIT-2.0.4.tar.gz"
-    sha256 "620fa4eb12375021bef6e4f237cbd2dd5d49e56beb414bee052c746beef1807d"
-  end
+    resource "luarocks" do
+      url "https://github.com/keplerproject/luarocks/archive/5d8a16526573b36d5b22aa74866120c998466697.tar.gz"
+      sha256 "cae709111c5701235770047dfd7169f66b82ae1c7b9b79207f9df0afb722bfd9"
+    end
 
-  resource "luarocks" do
-    url "https://github.com/keplerproject/luarocks/archive/0587afbb5fe8ceb2f2eea16f486bd6183bf02f29.tar.gz"
-    sha256 "c8ad50938fed66beba74a73621d14121d4a40b796e01c45238de4cdcb47d5e0b"
-  end
+    resource "unibilium" do
+      url "https://github.com/mauke/unibilium/archive/v1.2.0.tar.gz"
+      sha256 "623af1099515e673abfd3cae5f2fa808a09ca55dda1c65a7b5c9424eb304ead8"
+    end
 
-  resource "unibilium" do
-    url "https://github.com/mauke/unibilium/archive/v1.1.4.tar.gz"
-    sha256 "8b8948266eb370eef8100f401d530451d627a17c068a3f85cd5d62a57517aaa7"
-  end
+    resource "libtermkey" do
+      url "http://www.leonerd.org.uk/code/libtermkey/libtermkey-0.18.tar.gz"
+      sha256 "239746de41c845af52bb3c14055558f743292dd6c24ac26c2d6567a5a6093926"
+    end
 
-  resource "libtermkey" do
-    url "http://www.leonerd.org.uk/code/libtermkey/libtermkey-0.18.tar.gz"
-    sha256 "239746de41c845af52bb3c14055558f743292dd6c24ac26c2d6567a5a6093926"
-  end
+    resource "libvterm" do
+      url "https://github.com/neovim/libvterm/archive/1b745d29d45623aa8d22a7b9288c7b0e331c7088.tar.gz"
+      sha256 "3fc75908256c0d158d6c2a32d39f34e86bfd26364f5404b7d9c03bb70cdc3611"
+    end
 
-  resource "libvterm" do
-    url "https://github.com/neovim/libvterm/archive/1b745d29d45623aa8d22a7b9288c7b0e331c7088.tar.gz"
-    sha256 "3fc75908256c0d158d6c2a32d39f34e86bfd26364f5404b7d9c03bb70cdc3611"
+    resource "jemalloc" do
+      url "https://github.com/jemalloc/jemalloc/releases/download/4.0.2/jemalloc-4.0.2.tar.bz2"
+      sha256 "0d8a9c8a98adb6983e0ccb521d45d9db1656ef3e71d0b14fb333f2c8138f4611"
+    end
   end
 
-  resource "jemalloc" do
-    url "https://github.com/jemalloc/jemalloc/releases/download/3.6.0/jemalloc-3.6.0.tar.bz2"
-    sha256 "e16c2159dd3c81ca2dc3b5c9ef0d43e1f2f45b04548f42db12e7c12d7bdf84fe"
+  head do
+    # Third-party dependencies for latest repo revision.
+    resource "libuv" do
+      url "https://github.com/libuv/libuv/archive/v1.7.3.tar.gz"
+      sha256 "db5d46318e18330c696d954747036e1be8e2346411d4f30236d7e2f499f0cfab"
+    end
```

Comment:
Thanks, but we don't want to do that, otherwise users might randomly get build errors if changes are pushed to the dependency repos.